app/src/layouts/common/sidebar/supportBlock/supportBlock.jsx

```diff
@@ -83,17 +85,21 @@ export const SupportBlock = ({ options }) => {
     trackEvent(
       HELP_AND_SUPPORT_EVENTS.clickOnSupportModalBtn(messages.requestSupport.defaultMessage),
     );
+    trackEvent(HELP_AND_SUPPORT_GA4_EVENTS.CLICK_REQUEST_SUPPORT_BUTTON);
```

Comment:
I see almost all events are tracked from the same places as old events.
So, why we can not change only properties of the HELP_AND_SUPPORT_EVENTS.clickOnSupportModalBtn and not create a new file with almost the same events?

Reply:
I deleted the old events and renamed and left the new ones so that the new events would not be in the same file with the old ones.